app/controllers/avo/actions_controller.rb

```diff
@@ -64,7 +64,7 @@ def handle
     private
 
     def action_params
-      params.permit(:authenticity_token, :resource_name, :action_id, :button, fields: {})
+      @action_params ||= params.permit(:id, :authenticity_token, :resource_name, :action_id, :button, fields: {})
```

Comment:
Memoizing and permitting `id`.

The `id` is present when an action is clicked from a show page.